# HG changeset patch
# User Rik <rik@octave.org>
# Date 1514312096 28800
#      Tue Dec 26 10:14:56 2017 -0800
# Branch stable
# Node ID 746081e3cbddcc80861c95fe62d5f1fd639c92a5
# Parent  b9d482dd90f32c78d544125ec063dbd88c6da476
doc: Delete extra ']' in scanf docstring (bug #52742).

* file-io.cc (Fscanf): Delete extra ']' in deftypefnx calling form.

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -1141,17 +1141,17 @@ character to be read is returned in @var
     }
 
   return retval;
 }
 
 DEFUN (scanf, args, nargout,
        doc: /* -*- texinfo -*-
 @deftypefn  {} {[@var{val}, @var{count}, @var{errmsg}] =} scanf (@var{template}, @var{size})
-@deftypefnx {} {[@var{v1}, @var{v2}, @dots{}, @var{count}, @var{errmsg}]] =} scanf (@var{template}, "C")
+@deftypefnx {} {[@var{v1}, @var{v2}, @dots{}, @var{count}, @var{errmsg}] =} scanf (@var{template}, "C")
 This is equivalent to calling @code{fscanf} with @var{fid} = @code{stdin}.
 
 It is currently not useful to call @code{scanf} in interactive programs.
 @seealso{fscanf, sscanf, printf}
 @end deftypefn */)
 {
   int nargin = args.length ();
 
